ai_agents/agents/ten_packages/extension/tencent_tts_python/extension.py

```
@@ -146,6 +145,12 @@ async def request_tts(self, t: TTSTextInput) -> None:
             self.ten_env.log_info(
                 f"Requesting TTS for text: {t.text}, text_input_end: {t.text_input_end} request ID: {t.request_id}",
             )
+            if self.client is None:
+                self.client = TencentTTSClient(
+                    self.config, self.ten_env, self.vendor()
+                )
+                asyncio.create_task(self.client.start())
```

Ruff (0.14.0)

Store a reference to the return value of `asyncio.create_task`

(RUF006)